docs/source/gen/flytectl_register_files.rst

```diff
@@ -160,5 +162,5 @@ Options inherited from parent commands
 SEE ALSO
 ~~~~~~~~
 
-* :doc:`flytectl_register` 	 - Register tasks/workflows/launchplans from a list of generated serialized files.
+* :doc:`flytectl_register` 	 - Register tasks, workflows, launch plans from a list of generated serialized files.
```

Comment:
is it launch plans or launchplans?

Reply:
launch plan when we are talking about it in general, and Launchplan when we are referring to a specific one..